src/components/inscripcionTurnos/InscripcionActividad.js

```diff
@@ -31,6 +32,7 @@ export default function Actividad() {
   const { create } = useApi('turnos');
   const usuario = useRecoilValue(usuarioState);
   const [iconoCargando, setIconoCargando] = useState(false);
+  const [errorAlPedirTurno, setErrorAlPedirTurno] = useState('');
```

Comment:
Si no hay valor inicial, mejor no poner nada.

```suggestion
  const [errorAlPedirTurno, setErrorAlPedirTurno] = useState();
```